Makefile

```diff
@@ -63,11 +63,23 @@ clean: header ## Cleanup the project binary (bin) folders
 	@echo "Cleaning harvest files"
 	@rm -rf bin
 
-build: clean deps harvest collectors exporters ## Build the project
+test: ## run tests
+	@echo "Running tests"
+	go test -v ./...
 
-package: clean deps build dist-tar ## Package Harvest binary
+fmt: ## format the go source files
+	@echo "Running gofmt"
+	@find ./ -type f -name "*.go" -exec gofmt -w {} \;
```

Comment:
`go fmt ./...` is quicker and more idiomatic

Reply:
done.